### DISCUSSION TITLE
AttributeError: 'dict' object has no attribute '_sa_instance_state'

### DISCUSSION THREAD

**Discussion by jrlopes2005** [sqlalchemy/sqlalchemy#7767] [Usage Questions]
Hello,

Im trying to create a SQLAlchemy model instance

The code breaks in this line: 
   obj = UserModel(**db_obj.dict())

db_obj looks like: 
{
  "name": "test",
  "age": 25,
  "groups": [
    {
      "name": "First group"
    }
  ]
}

Im getting : AttributeError: 'dict' object has no attribute '_sa_instance_state'.
 
Can someone help me? I think the problem is in the groups relationship

**Comment by CaselIT** [comment]
Hi,

Yes, you need to create orm instance for the nested relationship.
Please see this similar discussion https://github.com/sqlalchemy/sqlalchemy/discussions/7336

**Comment by jrlopes2005** [comment]
Thank you for the answer.

So, there is no way to map my pydantic request schema with nested fields into a SQLAlchemy model with relationships?

**Reply by CaselIT** [reply]
in the linked discussion there is a solution. also if I'm not mistaken pydantic can do something like this https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances
